common/src/main/java/com/wynntils/features/stateless/NotificationsFeature.java

```diff
@@ -2,16 +2,16 @@
  * Copyright © Wynntils 2022.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
-package com.wynntils.features.internal;
+package com.wynntils.features.stateless;
 
-import com.wynntils.core.features.InternalFeature;
+import com.wynntils.core.features.StatelessFeature;
 import com.wynntils.features.user.overlays.GameNotificationOverlayFeature;
 import com.wynntils.mc.utils.McUtils;
 import com.wynntils.wc.event.NotificationEvent;
 import net.minecraft.network.chat.TextComponent;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-public class NotificationsFeature extends InternalFeature {
+public class NotificationsFeature extends StatelessFeature {
```

Comment:
Maybe this can be solved similar to how I do NPC chats. Don't even send out the event if the Overlay is not active, but print it to the chat instead.

Reply:
done